cookiecutter/hooks.py

```diff
@@ -104,4 +110,4 @@ def run_hook(hook_name, project_dir, context):
     if script is None:
         logging.debug('No hooks found')
         return
-    return run_script_with_context(script, project_dir, context)
+    run_script_with_context(script, project_dir, context)
```

Comment:
well, it was meant as a cleanup, `run_script_with_context` was never returning anything, so I removed the return to make clear this function is a [command not a query](https://en.wikipedia.org/wiki/Command%E2%80%93query_separation).

But I can revert the change, if you have other plans or I missed something. :)